ctru-rs/src/services/mod.rs

```diff
@@ -34,6 +34,8 @@ cfg_if::cfg_if! {
             //! [package.metadata.cargo-3ds]
             //! romfs_dir = "romfs"
             //! ```
+
+            compile_error!("romfs feature is enabled but no romfs found!");
```

Comment:
Won't this fail compilation for any program not using ROMFS?

Reply:
Yes, it would have. It's been fixed.